docs/resources/access_control_rule_set.md

```diff
@@ -138,10 +138,138 @@ resource "databricks_access_control_rule_set" "automation_sp_rule_set" {
 }
 ```
 
+## Group rule set usage
+
+Through a Databricks workspace:
+
+```hcl
+locals {
+  account_id = "00000000-0000-0000-0000-000000000000"
+}
+
+// account level group
+data "databricks_group" "ds" {
+  display_name = "Data Science"
+}
+
+data "databricks_user" "john" {
+  user_name = "john.doe@example.com"
+}
+
+resource "databricks_access_control_rule_set" "ds_group_rule_set" {
+  name = "accounts/${local.account_id}/groups/${databricks_group.ds.id}/ruleSets/default"
+
+  grant_rules {
+    principals = [data.databricks_user.john.acl_principal_id]
+    role       = "roles/group.manager"
+  }
+}
+```
+
+Through AWS Databricks account:
+
+```hcl
+locals {
+  account_id = "00000000-0000-0000-0000-000000000000"
+}
+
+// initialize provider at account-level
+provider "databricks" {
+  host       = "https://accounts.cloud.databricks.com"
+  account_id = local.account_id
+  username   = var.databricks_account_username
+  password   = var.databricks_account_password
```

Comment:
Let show examples with client_id/secret instead of username/password.

Reply:
are there any examples i can follow?

Reply:
https://registry.terraform.io/providers/databricks/databricks/latest/docs#special-configurations-for-aws - for AWS